infrastructure/cdn-in-a-box/traffic_monitor/run.sh

```diff
@@ -103,10 +103,10 @@ export TO_PASSWORD=$TO_ADMIN_PASSWORD
 
 touch /opt/traffic_monitor/var/log/traffic_monitor.log
 
-# Do not start until there is a valid CRConfig available
-until [ $(to-get "/CRConfig-Snapshots/$CDN_NAME/CRConfig.json" 2>/dev/null | jq -c -e '.config|length') -gt 0 ] ; do
-	echo "Waiting on valid CRConfig..."; 
-  	sleep 3; 
+# Do not start until there a valid Snapshot has been taken
+until [ $(to-get "/api/1.4/cdns/$CDN_NAME/snapshot" 2>/dev/null | jq -c -e '.config|length') -gt 0 ] ; do
```

Comment:
The responses of these 2 API are different so you also have to modify jq parameter:
```
"/api/1.4/cdns/$CDN_NAME/snapshot" |jq -c -e '.response.config|length'
```

by the way, not sure if this could be changed to new API:
[070-RASCAL-Traffic_Monitor.json](https://github.com/apache/trafficcontrol/blob/master/infrastructure/cdn-in-a-box/traffic_ops_data/profiles/070-RASCAL-Traffic_Monitor.json#L50)

Reply:
Well, given that they do have different responses, I don't think swapping them out in the TM config will work without modifying TM itself.